pkg/auth/manager/ocmshares/ocmshares.go

```diff
@@ -159,6 +159,8 @@ func (m *manager) Authenticate(ctx context.Context, ocmshare, sharedSecret strin
 		},
 	}
 
+	user.Opaque = utils.AppendJSONToOpaque(user.Opaque, "impersonating-user", userRes.RemoteUser)
```

Comment:
is the "impersonating-user" suitable for an anonymous user?

Reply:
Not sure what you mean.

"impersonating-user" can be 
* a hardcoded struct when this is a public link
* a remote user when this is a federate share

Reply:
Nevermind. This is just a naming nitpick.